zookeeper-server/src/main/java/org/apache/zookeeper/server/backup/BackupManager.java

```diff
@@ -278,9 +278,9 @@ public void run(int iterations) {
           }
         }
       } catch (InterruptedException e) {
-        logger.warn("Interrupted exception while waiting for backup interval.", e);
+        logger.warn("Interrupted exception while waiting for backup interval.", e.fillInStackTrace());
```

Comment:
What's the reason of using `e.fillInStackTrace()`?

Reply:
In our local deployments we found that without `e.fillInStackTrace()` it won't show the detailed stack trace of exceptions thrown in the lower stream logic which makes it hard to know why the exceptions occur.